moveit_core/collision_distance_field/src/collision_distance_field_types.cpp

```diff
@@ -52,8 +52,9 @@ collision_detection::determineCollisionSpheres(const bodies::Body* body, Eigen::
     const auto radius = body->getDimensions()[0];
```

Comment:
This radius doesn't consider padding and scaling. I guess that's what you wanted to consider?

Reply:
No need for scaling when it checks only for 0.0, but fair enough! :)